imports/plugins/included/payments-authnet/server/methods/authnet.js

```diff
@@ -98,9 +102,12 @@ Meteor.methods({
     } else {
       throw new Meteor.Error("invalid-transaction-type", "Invalid Transaction Type");
     }
-
-    const result =  Promise.await(authResult);
-    return result;
+    try {
+      const result =  Promise.await(authResult);
+      return result;
+    } catch (error) {
+      throw new Meteor.Error(400, error.message);
```

Comment:
So passing in this number here is deprecated. Please check the style guide but basically it should be in the format
`Meteor.Error("short-description", "Detailed description")`